### ISSUE TITLE
Bug when using oapmirror

### ISSUE THREAD

**Issue by mingzeng8** [cbersch/pst-optexp#5]
Hi,
I am using pst-optexp for experiment diagram. I found a problem when using oapmirror. The following code reproduce this:

```
\documentclass{article}
\usepackage{pst-optexp}
\begin{document}
	\begin{pspicture}[showgrid](8,2)
    	\pnode(5.0,0.9){In}
    	\pnode(1.0,1.0){OAP}
    	\pnode(7.0,1.0){F}
    	\oapmirror[compname=OAP1](In)(OAP)(F)
    	\drawwidebeam[beamwidth=.2](In){OAP1}(F)
	\end{pspicture}
\end{document}
```

The output is 
![image](https://user-images.githubusercontent.com/37693994/152529531-352eb74d-064d-4b47-aafa-ad9b70d27884.png)
which is not expected.

If the coordinate of {In} is changed to (5.0,1.0), nothing appears. If the coordinate of {In} is changed to (5.0,1.1), the result is correct. It seems this problem appears when the vertical coordinate of {In} is below {OAP}. Would you check this? Thanks a lot!

Ming

**Comment by cbersch** [comment]
@mingzeng8  Thank you for the report. I think I fixed it, you can try it from the branch. I must write some tests before merging it.

**Comment by cbersch** [comment]
Fixed in version 6.1
